pkgs/desktops/maxx/default.nix

```diff
@@ -81,7 +73,7 @@ in stdenv.mkDerivation {
     homepage = http://www.maxxinteractive.com;
```

Comment:
```suggestion
    homepage = http://maxxdesktop.arcadedaydream.com;
```

Reply:
@markuskowa thanks! Maxx.arcadedaydream.com contains only outdated pages. I've pointed to their facebook instead, which is their current news source